years_to_hours.rb

```diff
@@ -0,0 +1,31 @@
+def hours_in_year
+puts "Enter a number of years"  
+years = gets.chomp # this returns a string  
+years = years.to_i # this converts a string to an integer  
+hours = years * 365 * 24  
+puts "That's #{hours} hours."
+end
```

Comment:
Perfect with another style thing: I know this isn't Python so the program will still run without a problem, but for the humans reading it, it's good to add one level of indentation (2 spaces in Ruby) for the inside of a function.

```suggestion
def hours_in_year
  puts "Enter a number of years"  
  years = gets.chomp # this returns a string  
  years = years.to_i # this converts a string to an integer  
  hours = years * 365 * 24  
  puts "That's #{hours} hours."
end
```